documentation/CAMARA-API-access-and-user-consent.md

```diff
@@ -373,6 +377,186 @@ The Client Credentials grant type is used to obtain a 2-legged access_token that
+
+opt Access to API fails due to Access Token Expired
+  BE->>ExpO: Access Operator CAMARA API <br> Authorization: Bearer {OperatorAccessToken}        
+  ExpO->>ExpO: Decrypt and validate<br>Access Token. It has expired.  
+  ExpO->>BE: 401 Unauthorized<br>{"code": "UNAUTHENTICATED",<br> "message": "Token Expired"}
+end
+
+alt Standard OIDC Refresh Token flow between Invoker and API Exposure Platform
+  BE->>ExpO: POST /token<br>refresh_token={OperatorRefreshToken}
+  ExpO->>ExpO: Validate {OperatorRefreshToken}<br>fetch associated info (scopes)
+  ExpO->>ExpO: Check legal basis of the purpose<br>associated with the token<br> e.g.: contract, legitimate_interest, consent, etc
+  opt If User Consent is required for the legal basis of the purpose  
+      ExpO->>Consent: Check if Consent is granted    
+  end
+  alt If Consent is NOT granted (consent expired/revoked) - The flow fails
+    ExpO->>BE: 400 Bad Request<br>{"error":"invalid_scope"}        
+  else If Consent is Granted or Consent not needed for legal basis   
+    ExpO->>ExpO: Generate<br>new {NewOperatorAccessToken}<br> and {NewOperatorRefreshToken}
+    ExpO->>ExpO: Save new {NewOperatorRefreshToken}
+    ExpO->>BE: 200 OK <br> {NewOperatorAccessToken}<br>{NewOpeatorRefreshToken}
+  end 
+end
+alt Regular Access to API
+  BE->>ExpO: Access Operator CAMARA API <br> Authorization: Bearer {NewOperatorAccessToken}        
+  ExpO->>ExpO: Decrypts NewOperatorAccessToken,<br>grants Access,<br>progresses request to API Backend,<br>gets API response  
+  ExpO->>BE: CAMARA API Response
+  Note over BE,FE: Response
+end
+```
+
+**Key concepts**
+
+- Same considerations that apply for general call flows in [Authorization flows / grant types](#authorization-flows--grant-types) also apply for this flow.
+
+**Flow description**
+
+When a network feature is needed (step 1), the API invoker tries to consume a CAMARA API with an Access Token, but it has already expired, so it gets an error (steps 2-3). 
+
+Upon this error, the API invoker performs standard Refresh Token Grant flow, providing the Operator Refresh Token in order to get a new pair of Operator Refresh Token and Operator Access Token (step 4). 
+
+The Operator validates the refresh token and retrieves the related information, i.e.: scopes/purpose (step 5) and checks the legal basis of the purpose associated with the token (step 6). If the purpose requires the user's consent, the Operator validates whether the consent is granted (step 7). If the consent is NOT granted e.g., the consent has expired or the user has revoked the consent, the Operator will not issue a new token and will return an error (step 8) and the refresh token flow will fail. 
+
+If the user's consent is granted or not required for the applied legal basis, the Operator generates a new pair of Operator Access Token and Aggregator Access Token (step 9), saves the refresh token (step 10), and responds to the API invoker with this information (step 11). 
```

Comment:
This text can't be added because it adds `Aggregator Access Token` and `Operator Access Token` which are unknown concepts in Camara.

Reply:
@AxelNennker My bad, one of the things I did before moving the documentation was to remove any concept unknown to CAMARA like the aggregation model of GSMA Opengateway. Good catch, thanks. I will review the PR to correct it.

Reply:
Typo fixed in https://github.com/camaraproject/IdentityAndConsentManagement/pull/123/commits/354d61853ede084cc57d42e14e4affea02ad5078.